reflect_test.go

```diff
@@ -131,7 +131,7 @@ func TestReflect(t *testing.T) {
 	tt(t, func() {
 		// Testing dbgf
 		// These should panic
-		toValue("Xyzzy").toReflectValue(reflect.Ptr)
+		// toValue("Xyzzy").toReflectValue(reflect.Ptr)
```

Comment:
The entire test is disabled by the previous if (true) statement, but because of my changes it would no longer compile. I'm not sure what was the purpose of the test and why it was disabled, so I decided not to remove it.